websocket-tcp-chat/client.py

```diff
@@ -1,52 +1,60 @@
 #!/usr/bin/env python3
-"""websocket cmd client for wssrv.py example."""
+"""websocket cmd client for actix/websocket-tcp-chat example."""
 import argparse
 import asyncio
 import signal
 import sys
 
 import aiohttp
 
+queue = asyncio.Queue()
 
-def start_client(loop, url):
+
+async def start_client(url, loop):
     name = input('Please enter your name: ')
 
-    # send request
-    ws = yield from aiohttp.ClientSession().ws_connect(url, autoclose=False, autoping=False)
+    ws = await aiohttp.ClientSession().ws_connect(url, autoclose=False, autoping=False)
 
-    # input reader
     def stdin_callback():
         line = sys.stdin.buffer.readline().decode('utf-8')
         if not line:
             loop.stop()
         else:
-            ws.send_str(name + ': ' + line)
```

Comment:
not sure I understand, why you needt o put `ws.send_str` into a separate queue?

I guess it is not related to heartbeats itself?

Reply:
In the current implementation, it does not send messages. Since it does not solve the routine in the callback. I fixed it.

Reply:
https://docs.python.org/3/library/asyncio-eventloop.html?highlight=add_reader#asyncio.AbstractEventLoop.add_reader Callback can not be solved in future and `ws.send_str` is solved in the future so it does not solve any of them. I discovered it because the client in python was disconnected by hb timeout.

Reply:
is `send_str` must be awaitable?
I ghought it would be possible to use async callback here, but if not then ok

Reply:
I based myself on this post, https://stackoverflow.com/questions/29475007/python-asyncio-reader-callback-and-coroutine-communication . The asyncio documentation is little more than impossible

Reply:
OK thanks for explanation. I'm not really familiar with python async story 😄